source/extensions/filters/http/ip_tagging/ip_tagging_filter.cc

```diff
@@ -48,20 +48,15 @@ IpTaggingFilterConfig::IpTaggingFilterConfig(
                         entry.address_prefix(), entry.prefix_len().value()));
       }
     }
+
     tag_data.emplace_back(ip_tag.ip_tag_name(), cidr_set);
+    stat_name_set_->rememberBuiltin(absl::StrCat(ip_tag.ip_tag_name(), ".hit"));
```

Comment:
@jmarantz here we add each tag...

Reply:
Sorry; my bad. OK this looks great! Thanks!